### PR TITLE
homework 4

### PR DESCRIPTION
**Pull request by maintf**

### CODE REVIEW

File: Mykhaliuk/HW4/1.py [hunk 1]
```diff
@@ -0,0 +1,10 @@
+#The YEAR is known. Determine whether this year is intercalary and to what century this year belongs
+year = int(input('Enter a year:'))
+
+if year % 4 == 0 and (year % 100 != 0 or year % 400 == 0):
+    print(year, "is a leap year.")
+else:
+    print(year, "is not a leap year.")
+
+century = (year - 1) // 100 + 1
```
Comment (katemalash) [inline Mykhaliuk/HW4/1.py:9]: Why do you use year - 1 here?

File: Mykhaliuk/HW4/2.py [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+    sides = (input("Enter 2 side lengths with space in between: ").split())
+    perimetr = ((int(sides[0]) + int(sides[1])) *2)
+elif figure=='3':
+    sides = (input("Enter 3 side lengths with spaces in between: ").split())
+    perimetr = (int(sides[0]) + int(sides[1]) + int(sides[2]))
+elif figure=='4':
+    radius = int(input('enter the radius:'))
+    perimetr = 2 * pi * radius
+
+print('figure perimeter:', perimetr)
```
Comment (katemalash) [inline Mykhaliuk/HW4/2.py:20]: It was required to calculate square here, not perimeter.

File: Mykhaliuk/HW4/3.py [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+positivity = 'positive'
+
+if int(number) < 0:
+    positivity = 'negative'
+    number_of_didgits -= 1
+    print("you've entered", positivity, number_of_didgits, '- didgit number')
+elif number == '0':
+    print("you've entered zero")    
+else:
+    print("you've entered", positivity, number_of_didgits, '- didgit number')
```
Comment (katemalash) [inline Mykhaliuk/HW4/3.py:15]: Nice decision!

File: Mykhaliuk/HW4/5.py [hunk 4]
```diff
@@ -0,0 +1,14 @@
...
+
+print ("200:", amount // 200)
+amount -= ((amount // 200)*200)
+print ("100:", amount // 100)
+amount -= ((amount // 100)*100)
+print ("50:", amount // 50)
+amount -= ((amount // 50)*50)
+print ("10:", amount // 10)
+amount -= ((amount // 10)*10)
+print ("1:", amount // 1)
```
Comment (katemalash) [inline Mykhaliuk/HW4/5.py:14]: This is working Way

File: Mykhaliuk/HW4/4.py [hunk 5]
```diff
@@ -0,0 +1,102 @@
...
+squareSideButton = Button(root, text="Create square", command=square)
+squareSideButton.grid(row=4,column=0)
+
+resaultButton = Button(root, text='Resault', command=resault)
+resaultButton.grid(row=4,column=1)
+
+clearButton = Button(root, text="Clear", command=clear)
+clearButton.grid(row=7,column=1)
+
+root.mainloop()
```
Comment (katemalash) [inline Mykhaliuk/HW4/4.py:102]: Perfect Way

**Review by katemalash — CHANGES_REQUESTED** [review]
Please fix task 2

File: Mykhaliuk/HW4/1.py (hunk 1 above)
Comment (maintf) [reply Mykhaliuk/HW4/1.py:9]: so that year 2000 still belongs to 20th century

Comment (katemalash) [reply Mykhaliuk/HW4/1.py:9]: Nice)

File: Mykhaliuk/HW4/2.py [hunk 6]
```diff
@@ -1,20 +1,21 @@
...
+    p = (int(sides[0]) + int(sides[1]) + int(sides[2]))/2
+    square = sqrt(p*(p-int(sides[0]))*(p-int(sides[1]))*(p-int(sides[2])))
 elif figure=='4':
     radius = int(input('enter the radius:'))
-    perimetr = 2 * pi * radius
+    square = pi * radius**2
 
-print('figure perimeter:', perimetr)
\ No newline at end of file
+print('figure square:', square)
```
Comment (katemalash) [inline Mykhaliuk/HW4/2.py:21]: Good Now!

**Review by katemalash — APPROVED** [review]
Now you can merge this pull-request

**Pull request merged**